src/subscribers.ts

```diff
@@ -4,4 +4,12 @@ export let subscribers: ISubscriber[] = [];
 
 export function clearSubscribers() {
   subscribers = [];
+}
+
+export function removeSubscriberByAction(action: string) {
+  subscribers = subscribers.filter(subscriber => subscriber.action !== action);
+}
+
+export function removeSubscriberByCallback(callback: (args: any[]) => void) {
+  subscribers = subscribers.filter(subscriber => subscriber.callback !== callback)
```

Comment:
Is this the right approach here?

Reply:
I guess this should work but not 100% sure. We could go with and see if it works since we still lack some decent tests in this project.